### ISSUE TITLE
Adaptive Gradient Clipping instead of PyTorch Norm Clipping

### ISSUE THREAD

**Issue by Handagot** [Handagot/DeepLearningShogi#1]

**Comment by Handagot** [comment]
use built-in optimizer instead
